### PR TITLE
Sussus amogus

### PR DESCRIPTION
**Pull request by Ohagi-Chan**
## About The Pull Request
Adds the crewmembers, from critically acclaimed videogame phenomenon "Among Us"
The crewmembers must discover which of them is the imposter and vote the imposter out before it can kill them all.

Imposters are incredibly dangerous, possessing much greater lethality, vent crawling and the ability to change costumes.

## Why It's Good For The Game
Red is sus!

to do:

- [ ] complete body reporting logic
- [ ] give the crew jobs to do?

## Changelog
:cl:
add: adds amogi
/:cl:

red_amogus
red_amogus
amogus_change

### CODE REVIEW

**Comment by Ohagi-Chan** [comment]
Why the fuck did I make this?
Had to teach myself how to read antag code for a meme mob that terra var edited in 2 minutes.

**Comment by Ohagi-Chan** [comment]
On second thought, just because Terra gabe the imposter 1000 hp, doesn't mean that it's a good idea.
When Terra made an imposter, it's goal was to fight the crew, not be defeated by the "crewmembers"
1000 hp when they have 15 melee damage is WAY too much.

**Comment by TheFakeElon** [comment]
licensing for the icons?

File: austation/code/modules/mob/living/simple_animal/hostile/amogus.dm [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+/mob/living/simple_animal/hostile/stickman/amogus/Initialize(mapload, wizard_summoned)
+	. = ..()
+	var/mob/M = src
+	if(!length(available_colours))  //  We ran out of colours boss, what do we do; start a new set? I'd rather not just prevent the mob from being spawned.
+		available_colours = amo_colours
+	icon_state = pick(available_colours)
+	available_colours &= ~icon_state
+
+	if(!is_sus && sus && rand() <= 0.7)  //  Don't create a new antag if all three are true:
```
Comment (TheFakeElon) [inline austation/code/modules/mob/living/simple_animal/hostile/amogus.dm:103]:
```suggestion
	if(!is_sus && sus && prob(70))  //  Don't create a new antag if all three are true:
```

**Comment by Ohagi-Chan** [comment]
> licensing for the icons?

@TheFakeElon 
Oof, you're absolutely right.
Should I just cancel the PR?

**Comment by TheFakeElon** [comment]
uhhh yeah, probably. I highly doubt we'd ever get in trouble for using it but legally speaking we would be in the wrong, so it's probably best to close unless you want to make your own parody sprites

**Comment by Ohagi-Chan** [comment]
At times I wondered why the hell I was even making this ridiculous PR, but all the same I don't really regret the experience.
I've learned a little bit about how antag teams are coded and maybe next time I dabble in antags I'll be a bit more confident.

I'm happy to close this absurd, self-referencial PR.

**Pull request closed without merging**